libs/checkbox/src/lib/checkbox/checkbox.component.html

```diff
@@ -0,0 +1 @@
+<input type="checkbox" [checked]='checked' [disabled]='disabled' (change)='checkedChange.emit(!checked)'><ng-content></ng-content>
```

Comment:
In `change` event it is better to use `$event.target.checked` value instead of negation to avoid de-sync of actual and internal states.